lib/miq_expression.rb

```diff
@@ -678,7 +685,7 @@ def self.quote_human(val, typ)
     when "string", "date", "datetime"
       return "\"#{val}\""
     else
-      return quote(val, typ)
+      quote(val, typ)
```

Comment:
Yeah I thought this too....I kind of had blinders on only addressing the ones the cop found, but I'll nix these too